src/Psy/Configuration.php

```diff
@@ -1251,4 +1253,24 @@ public function getStartupMessage()
     {
         return $this->startupMessage;
     }
+
+    /**
+     * Get the force array indexes.
+     *
+     * @return bool
+     */
+    public function getForceArrayIndexes()
```

Comment:
We're not 100% consistent, but most Boolean getters in Configuration don't use `get` prefix. Let's call this method `forceArrayIndexes`.